test/clj/nnts2/note/functional_test.clj

```diff
@@ -0,0 +1,25 @@
+(ns nnts2.note.functional-test
+  (:require  [clojure.test :refer :all]
+             [nnts2.fixtures :as fixtures]
+             [nnts2.note.handler :refer [create]]
+             [nnts2.note.db :refer [add]]))
+
+
+(use-fixtures :each fixtures/clear)
+(use-fixtures :once fixtures/setup fixtures/adduser)
+
+
+(deftest create-note-success
+  (testing "String title, content and valid user id -> note success"
```

Comment:
Consider writing simple, generic user level test strings rather than domain level ones, except for edge cases. For example, this could be converted to `should create note successfully for valid params`. This is only a convention of course, so feel free to ask around to get an idea of how folks write them.

Reply:
Ok. should I be writing more tests here?

Reply:
This seems to be testing both the note creation as well as the handler. Maybe separate those two out into different tests? Also, I'm following a naming convention similar to what Kiran mentioned. Let's stick to that?

Reply:
yes